src/test/java/io/vertx/core/logging/JULLogDelegateTest.java

```diff
@@ -40,10 +41,24 @@
   public static void initialize() throws IOException {
     // Clear value.
     System.clearProperty("vertx.logger-delegate-factory-class-name");
+    LoggerFactory.initialise();
```

Comment:
@cescoffier with re-initialization of the logger factory and removal of ignore annotations, the tests pass on my box, even as part of the suite. They didn't pass when the call to initialize was missing. I miss some context to make sure you had the same issue on CI, but if you can confirm it was something like:

```
Running io.vertx.core.logging.JULLogDelegateTest
[main] ERROR my-jul-logger - hello
[main] INFO my-jul-logger - hello
[main] WARN my-jul-logger - hello
Tests run: 4, Failures: 4, Errors: 0, Skipped: 0, Time elapsed: 0.015 sec <<< FAILURE! - in io.vertx.core.logging.JULLogDelegateTest
testError(io.vertx.core.logging.JULLogDelegateTest)  Time elapsed: 0.009 sec  <<< FAILURE!
java.lang.AssertionError: null
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at io.vertx.core.logging.JULLogDelegateTest.testError(JULLogDelegateTest.java:119)

testInfo(io.vertx.core.logging.JULLogDelegateTest)  Time elapsed: 0.004 sec  <<< FAILURE!
java.lang.AssertionError: null
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at io.vertx.core.logging.JULLogDelegateTest.testInfo(JULLogDelegateTest.java:65)

testDelegateUnwrap(io.vertx.core.logging.JULLogDelegateTest)  Time elapsed: 0 sec  <<< FAILURE!
java.lang.AssertionError: Unexpected unwrapped type: org.apache.logging.log4j.core.Logger cannot be cast to java.util.logging.Logger
	at org.junit.Assert.fail(Assert.java:88)
	at io.vertx.core.logging.JULLogDelegateTest.testDelegateUnwrap(JULLogDelegateTest.java:55)

testWarning(io.vertx.core.logging.JULLogDelegateTest)  Time elapsed: 0.002 sec  <<< FAILURE!
java.lang.AssertionError: null
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at io.vertx.core.logging.JULLogDelegateTest.testWarning(JULLogDelegateTest.java:172)


Results :

Failed tests: 
  JULLogDelegateTest.testDelegateUnwrap:55 Unexpected unwrapped type: org.apache.logging.log4j.core.Logger cannot be cast to java.util.logging.Logger
  JULLogDelegateTest.testError:119 null
  JULLogDelegateTest.testInfo:65 null
  JULLogDelegateTest.testWarning:172 null
```

then I'll push a last commit.